src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -84,6 +92,28 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
+    private _initializeV2(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        return this._klarnaScriptLoader.loadPayments()
+            .then(klarnaPayments => { this._klarnaPayments = klarnaPayments; })
+            .then(() => {
+                this._unsubscribe = this._store.subscribe(
+                    state => {
+                        if (state.paymentStrategies.isInitialized(options.methodId)) {
```

Comment:
I don't think you'll need this check at all if you just move the entire `subscribe` call just above `.then(() => this._store.getState())` (after the initial `_loadPaymentsWidget` call). That'll effectively say only react to changes to the `outstandingBalance` field after the initial loading of the widget.